release notes/v0.38.0.md

```diff
@@ -0,0 +1,180 @@
+k6 v0.38.0 is here! 🎉
+
+## New Features!
+
+### AWS JSLib
+
+There's a new addition to the officially supported k6 JavaScript libraries: [k6-jslib-aws](https://github.com/grafana/k6-jslib-aws).
+This library lets users interact with a selection of AWS services directly from their scripts. The library currently implements support for the [S3](https://k6.io/docs/javascript-api/jslib/aws/s3client/) and the [Secrets Manager](https://k6.io/docs/javascript-api/jslib/aws/secretsmanagerclient/) services.
+
+[The AWS JS lib documentation](https://k6.io/docs/javascript-api/jslib/aws) has examples and details on how to use the library in your scripts.
+
+### Accessing the consolidated and derived options from the default function ([#2493](https://github.com/grafana/k6/pull/2493))
+
+The `k6/execution` core module now lets you access the [consolidated and derived](https://k6.io/docs/using-k6/options/#where-to-set-options) options that k6 computed before it started executing the test. You can access consolidated options through the `exec.test.options` property. Note that consolidated options are frozen and cannot be modified. [The k6 execution module's documentation](https://k6.io/docs/javascript-api/k6-execution/#test) has examples and details on how to use the functionality in your scripts. 
+
+```javascript
+import exec from 'k6/execution';
+
+export const options = {
+  vus: 10,
+  duration : '30s'
+};
+
+export default function() {
+  console.log(exec.test.options.scenarios.default.vus) // 10
+}
+```
+
+### Dumping SSL keys to an NSS formatted key log file ([#2487](https://github.com/grafana/k6/pull/2487))
+
+This release adds the ability to dump SSL keys while making TLS connections.
+You then can use these keys to decrypt all traffic that k6 generates. 
+
+To accomplish this, set the `SSLKEYLOGFILE` environment variable to some file path and run k6.
+This will populate the file with the keys.
+Then you can use Wireshark to capture the traffic, decrypt it, and use that for debugging. 
+
+[Here's an example that uses curl to inspect TLS traffic](https://daniel.haxx.se/blog/2018/01/15/inspect-curls-tls-traffic/).
+
+###  Tagging metric values with the current scenario stage
+
+With the new consolidated script options, we've added a few helper functions to the [k6-jslib-utils](https://k6.io/docs/javascript-api/jslib/utils) library. You can use them to automatically tag all the emitted metric samples by k6 with the currently running stage.
+
+## Breaking Changes
+
+### `console` methods now pretty print objects and arrays ([2375](https://github.com/grafana/k6/pull/2375))
+
+For convenience, all `console` methods such as `console.log()` and `console.info()` will now automatically `JSON.stringify()` objects and arrays passed to it. Thus, instead of `console.log({'foo': 'bar'})` printing `[object Object]`, it will now print `{'foo': 'bar'}`, which will make the experience of debugging k6 scripts easier and more intuitive. 
```

Comment:
```suggestion
For convenience, all `console` methods such as `console.log()` and `console.info()` will now automatically `JSON.stringify()` objects and arrays passed to them. Thus, instead of `console.log({'foo': 'bar'})` printing `[object Object]`, it will now print `{'foo': 'bar'}`, which will make the experience of debugging k6 scripts easier and more intuitive. 
```